app/code/Magento/Theme/Model/ResourceModel/Theme/Grid/Collection.php

```diff
@@ -5,11 +5,62 @@
  */
 namespace Magento\Theme\Model\ResourceModel\Theme\Grid;
 
+use Magento\Theme\Model\ResourceModel\Theme\Collection as ThemeCollection;
+use Magento\Framework\Api\Search\SearchResultInterface;
+use Magento\Framework\Api\Search\AggregationInterface;
+
 /**
  * Theme grid collection
  */
-class Collection extends \Magento\Theme\Model\ResourceModel\Theme\Collection
+class Collection extends ThemeCollection implements SearchResultInterface
 {
+    /**
+     * @var AggregationInterface
+     */
+    protected $aggregations;
```

Comment:
Please change protected to private due to Magento [Technical Guidelines](https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html)